resources/user-agents/browsers/android-webview/android-webview-1-5-2-1.json

```diff
@@ -812,15 +617,6 @@
             "Android_4_2"
           ]
         },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*kthml*like*gecko*)*Version/#MAJORVER#.#MINORVER#*Chrome*Safari*",
```

Comment:
This one I ended up changing in the tests so that the Samsung Browser changes pick it up. I don't believe this is a real UA, as I found the UA used on this page: http://developer.samsung.com/technical-doc/view.do?v=T000000203

>Mozilla/5.0 (Linux; Android 4.4.4; en-au; SAMSUNG SM-N915G Build/KTU84P) AppleWebKit/537.36 (KTHML, like Gecko) Version/2.0 Chrome/34.0.1847.76 Mobile Safari/537.36

I assume it's a type-o on Samsung's page. I searched our logs for UAs with this format (KTHML instead of KHTML) and the only UA I found over the past year was the one above (assuming it got picked up by bots/traffic generators).